src/main/java/roomescape/member/Member.java

```diff
@@ -38,6 +39,10 @@ public String getPassword() {
     }
 
     public String getRole() {
-        return role;
+        return role.name();
+    }
+
+    public boolean isAdmin() {
+        return role.name().equals("ADMIN");
```

Comment:
이 부분에서 결국 String으로 "ADMIN" 을 선언해서 Role을 비교 하는거라면, enum을 선언하는게 의미가 있나 했어요.
말씀대로 확장성이나 안전성을 위해 enum을 사용 했다면,
```suggestion
    public boolean isAdmin() {
        return role == Role.ADMIN;
```
이런식으로 활용 해야 할 것 같아서요!